src/Template/GoldenEye.WebApi.SimpleDDD/Backend/Module.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using Contracts.Issues.Commands;
+using Contracts.Issues.Queries;
+using Backend.Issues;
+using Backend.Issues.Handlers;
+using Backend.Issues.Projections;
+using GoldenEye.Backend.Core.DDD.Registration;
+using GoldenEye.Backend.Core.Marten.Events.Storage;
+using GoldenEye.Backend.Core.Marten.Registration;
+using GoldenEye.Shared.Core.Modules;
+using Marten;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using IssueContracts = Contracts.Issues;
+
+namespace Backend
+{
+    public class Module : ModuleBase
```

Comment:
Please merge laterst master, currently `ModuleBase` was renamed to `Module`. See more in https://github.com/oskardudycz/GoldenEye/pull/58/files#diff-ebb3e38110765db415500e25d6cc4482